src/pages/AppList/AppListPage.tsx

```diff
@@ -3,21 +3,42 @@ import { Breadcrumb } from 'patternfly-react';
 import { ListPagesHelper } from '../../components/ListPage/ListPagesHelper';
 import AppListContainer from './AppListComponent';
 import { AppListFilters } from './FiltersAndSorts';
+import MetricsDuration from '../../components/MetricsOptions/MetricsDuration';
 
-const AppListPage: React.SFC<{}> = () => {
-  return (
-    <>
-      <Breadcrumb title={true}>
-        <Breadcrumb.Item active={true}>Applications</Breadcrumb.Item>
-      </Breadcrumb>
-      <AppListContainer
-        pagination={ListPagesHelper.currentPagination()}
-        currentSortField={ListPagesHelper.currentSortField(AppListFilters.sortFields)}
-        isSortAscending={ListPagesHelper.isCurrentSortAscending()}
-        rateInterval={ListPagesHelper.currentDuration()}
-      />
-    </>
-  );
+type AppListPageProps = {};
+
+type AppListPageState = {
+  rateInterval: number;
```

Comment:
As discussed in the other Jira, the duration used here should likely be synced with the Overview and Graph pages because it is being used to determine health of the list elements.  As such this should likely be using the redux value.